python/paddle/fluid/tests/unittests/test_dist_base.py

```diff
@@ -293,18 +282,17 @@ def _run_local(self,
         if check_error_log:
             err_log.close()
 
-        sys.stderr.write('local_stdout: %s\n' % local_ret)
+        sys.stderr.write('local_stdout: %s\n' % pickle.loads(local_ret))
         sys.stderr.write('local_stderr: %s\n' % local_err)
 
-        local_losses = local_ret.split("\n")
-        return local_losses
+        # local_losses = local_ret.split("\n")
+        return pickle.loads(local_ret)
 
     def _run_cluster(self, model, envs, check_error_log):
         # Run dist train to compare with local results
         ps0, ps1, ps0_pipe, ps1_pipe = self.start_pserver(model,
                                                           check_error_log, envs)
-        self._wait_ps_ready(ps0.pid)
-        self._wait_ps_ready(ps1.pid)
```

Comment:
no need to wait now, so 4 processes can start at the same time to reduce total test time.